### ISSUE TITLE
White line under header

### ISSUE THREAD

**Issue by itbel** [themeetinghouse/mobile#155]
I noticed that with the last merge there is now a white line under the header on the homescreen. I just wanted to check that this was an intended change as it is not on the approved design.

![Screenshot!UNITO-UNDERSCORE!20201127-162434!UNITO-UNDERSCORE!Expo](https://user-images.githubusercontent.com/42515854/100485446-104fe600-30ce-11eb-9328-ca86c75daa61.jpg)

┆Issue is synchronized with this [Wrike Task](https://www.wrike.com/open.htm?id=601686853)

**Comment by GeorgeBellTMH** [comment]
I think we should fix this...

**Comment by JonEsparaz** [comment]
If this is changed, I recommend ensuring consistency throughout the app.

I added the bottom border to the [Header component](https://github.com/themeetinghouse/mobile/blob/dev/components/Header/Header.tsx) for the sake of consistency. See below screen shots:

<img src=https://user-images.githubusercontent.com/48423418/100921720-f0903780-34aa-11eb-8949-aec20e72ea74.png width=200>
<img src=https://user-images.githubusercontent.com/48423418/100921723-f0903780-34aa-11eb-9e36-27c7a89a2527.png width=200>

...on iOS the bottom border is visible on _all_ headers (I'm not sure if things look different on Android). 

TL;DR: If you remove the bottom border on the homescreen, remove it everywhere.

**Comment by JonEsparaz** [comment]
Also, if you look carefully at the Figma mockups, there's a 1px `#1A1A1A` drop shadow on all headers. To conform precisely to the mockups, the easiest implementation is a hairline width bottom border (which is what React Navigation and our Header component do), but the color needs to be adjusted.

**Comment by GeorgeBellTMH** [comment]
I'm ok with the white line if we think it is helpful for dividing out the content a bit better. 

We might look at the location dropdown being not quite accurately vertically aligned in the first screenshot which makes it look weird.

**Comment by lucastbelem** [comment]
Hey team,

If we can just adjust the colour, we may be in good shape. It's ever so faint, but there is something in figma. Whatever gets us closest to the design without screwing up practical components of this experience.
